docs/data/material/migration/upgrade-to-v7/upgrade-to-v7.md

```diff
@@ -41,14 +41,7 @@ If you're using any of the following packages, they should remain unchanged duri
 - `@mui/x-tree-view`
 - `@mui/x-tree-view-pro`
 
-## Supported browsers and versions
-
-### Minimum React version
-
-The minimum supported version of React is v17.0.0 (the same as v6).
-If you want to upgrade React, it's recommended to finish upgrading Material UI first, then upgrade React.
-
-### Minimum TypeScript version
```

Comment:
I was so confused by this. I skim read headers, I saw `Minimum React version` so I assumed that the version changed. And no, actually it didn't. Removing this seems much better. I don't understand why we have it.